index.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable array-callback-return */
```

Comment:
- [ ] You did a great job working on your Js code! 🥇 However, please avoid disabling linter rules. Linters are important to ensure a consistent format and best practices all over your codebase. Please consider removing this line and fixing the rule! 🤓 🛠️

Reply:
Thanks for the advice. I will remove it and find the way to run the project without errors